frame/lottery/src/lib.rs

```diff
@@ -442,12 +455,12 @@ impl<T: Config> Pallet<T> {
 						Error::<T>::AlreadyParticipating
```

Comment:
Using a `BoundedSet` here would make matters easier, but I'm not sure if it is worth the refactor rn.

Reply:
can you elaborate?